source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -175,6 +175,7 @@ mx::DocumentPtr Graph::loadDocument(mx::FilePath filename)
     {
         if (!filename.isEmpty())
         {
+            std::cout << ">>> Reading file " << filename.asString() << std::endl;
```

Comment:
Thanks for adding this. I agree with your issue that logging should be in the UI, but better than nothing I guess.

Reply:
Ideally validation could be visualized in the UI yes but at least with a little log like this we know for sure which file did cause the errors :)